gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
@tmortagne I fixed the build by downgrading Build tools and gradle plugin version. It was failing due latest versions and mismatch of configurations on server.
Today, I setup Jenkins on my own server to test the fork repository. It works fine there, and build passes after setting SDK. You can see [here](http://139.59.38.89:8080/job/xwfork/29/).